### PR TITLE
Podspec preparation 

### PR DESCRIPTION
**Pull request by bobgodwinx**
- [x] CI fix + `3.10.2` release preparation

### CODE REVIEW

**Comment by freak4pc** [comment]
Let's try to give PRs better titles - it's a bit hard when everything is called develop :)

File: circle.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
       XCODE_TEST_REPORTS: /tmp/xcode-test-results
       LANG: en_US.UTF-8
     macos:
-      xcode: '10.1.0'
+      xcode: '10.0.0'
```
Comment (freak4pc) [inline circle.yml]: I don't see a reason to take this down to 10.0 , the failure doens't seem related to the xcode version. I just reran master and it passed with no issues: https://circleci.com/gh/RxSwiftCommunity/Action/85

Comment (bobgodwinx) [reply circle.yml]: This is really strange.. I will try it again let see if it passes correctly

**Comment by bobgodwinx** [comment]
All good now..

File: Action.podspec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Action"
-  s.version      = "3.10.0"
+  s.version      = "3.10.2"
```
Comment (freak4pc) [inline Action.podspec]: Shouldn't this be `3.10.1` ? 
```suggestion
  s.version      = "3.10.1"
```

File: Action.podspec [hunk 3]
```diff
@@ -20,8 +20,8 @@ Pod::Spec.new do |s|
   s.source_files  = "Sources/**/*.{swift}"
 
   s.frameworks  = "Foundation"
-  s.dependency "RxSwift", "~> 4.3"
-  s.dependency "RxCocoa", "~> 4.3"
+  s.dependency "RxSwift", "~> 4.0"
+  s.dependency "RxCocoa", "~> 4.0"
```
Comment (freak4pc) [inline Action.podspec]: Do we have nothing that relays on newer APIs in 4.3 etc ? e.g. Relay, deprecation of Variable, stuff like that?

File: Action.podspec (hunk 2 above)
Comment (bobgodwinx) [reply Action.podspec]: Nop.. I checked and `3.10.1` is already there. it was make for Carthage releases.. So this is why I choose the next in order to do a release that covers both Carthage and Cocoapods

File: Action.podspec (hunk 3 above)
Comment (freak4pc) [reply Action.podspec]: Well I guess if we would, CI wil fail

Comment (bobgodwinx) [reply Action.podspec]: Nop.. we're using the `InputSubject` and it's totally free from those family of `Relay`

**Pull request merged**